packages/core/src/indexing/postgres/migrations/2-mid-schema-verification.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
this isn't really a "migration" and doesn't feel like it belongs in this directory.  Can you move this file up a dir (and rename it)?

Reply:
can do - I thought that its going to be part of migration going forward e.g. once the table structure changes